src/browser/provider/built-in/remote.js

```diff
@@ -7,14 +7,19 @@ export default class RemoteBrowserProvider extends BrowserProviderBase {
     constructor () {
         super();
 
-        // NOTE: This can be used to disable resize correction when running unit tests.
-        this.disableResizeHack = false;
+        // NOTE: This can be used to disable resize correction or maximum size calculation when running unit tests.
+        this.disableHacks = false;
```

Comment:
It's not hacks anymore, since I can call `calculateMacSizeLimits` a hack. We need better name for this one \cc @VasilyStrelyaev